Probesys/bileto
# imp: Add navigation to organizations to the main menu

## Related issue(s)



https://github.com/Probesys/bileto/issues/580

## How to test manually



1. login with a user with the permission to create organizations (admin) and an operational role with all permissions, associated to a specific organization (i.e. they should not have access to all the organizations)
2. on the home page, check that the organizations no longer appear
1. check that you see a new tab in the navigation menu named "Organizations" and click on it
1. check that you see only the organizations you have access to, and that they appear as cards
1. check that you're allowed to create a new organization
1. check that you can open an organization to access its tickets and contracts
1. check that you can manage the organization in the new "settings" page

## Checklist
